doc/submitting-changes.xml

```diff
@@ -194,36 +187,17 @@ Additional information.
    </listitem>
    <listitem>
     <para>
-     Create pull request:
-     <itemizedlist>
-      <listitem>
-       <para>
-        Write the title in format <command>(pkg-name | nixos/&lt;module>): improvement</command>.
-        <itemizedlist>
-         <listitem>
-          <para>
-           If you update the pkg, write versions <command>from -> to</command>.
-          </para>
-         </listitem>
-        </itemizedlist>
-       </para>
-      </listitem>
-      <listitem>
-       <para>
-        Write in comment if you have tested your patch. Do not rely much on <command>TravisCI</command>.
-       </para>
-      </listitem>
-      <listitem>
-       <para>
-        If you make an improvement, write about your motivation.
-       </para>
-      </listitem>
-      <listitem>
-       <para>
-        Notify maintainers of the package. For example add to the message: <command>cc @jagajaga @domenkozar</command>.
-       </para>
-      </listitem>
-     </itemizedlist>
+     Create the pull request
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Follow <link xlink:href="https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes">the contribution guidelines</link>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Notify maintainers of the package. For example add to the message: <command>cc @jagajaga @domenkozar</command>.
```

Comment:
If you edit an existing package, ofBorg will notify the maintainers automatically, so maybe drop that?

Reply:
not yet. right @Lassulus  ?

Reply:
I think only the maintainer which are part of the organization. I'm not sure if this includes everyone by now?

Reply:
Also the cc part is still in the template. I looked at some pull-requests and it seems that ofBorg adds reviewers, but not all maintainers.

Reply:
Asked in nixos/rfcs#39 — it looks like vast majority of maintainers are currently indeed organisation members (via readonly maintainers team) and can be review-requested. If every maintainer has already been invited, we could drop the cc line in template too, I guess.

Reply:
So I wouldn't do it now. I'll come back to it in another pull-request.

Reply:
I think NixOS/RFCs#39 has been now implemented as well as it can be…

Reply:
removed the line from this pull-request. I don't have the privileges to change the pull-request template. @Mic92 @Lassulus ?